dbus/src/arg/mod.rs

```diff
@@ -86,11 +87,13 @@ fn ffi_iter() -> ffi::DBusMessageIter {
 
 /// An RAII wrapper around Fd to ensure that file descriptor is closed
 /// when the scope ends.
+#[cfg(unix)]
```

Comment:
As for Ownedfd, it's about to be deprecated in favor of using `std::fs::File` directly. That would be a backwards incompatible change though. If I refactored it into  `pub struct OwnedFd(std::fs::File)`, would that make life easier for you?
Then the struct would not have to be cfged on Windows.

Reply:
If you refactor it, seems that I'll have to resolve a lot of conflicts :)

> Then the struct would not have to be cfged on Windows

Now it's just empty on Windows